chapters/welcome.ipynb

```diff
@@ -29,28 +29,69 @@
     "docker compose up\n",
     "```\n",
     "\n",
-    "The book's code examples work with many different search engines and vector databases. By default, the book uses Apache Solr as the primary storage, matching, and ranking engine, but you can easily swap in your your favorite engine by running the `set_engine` function here: [engines/supported-engines.ipynb](engines/supported-engines.ipynb).\n",
-    "\n",
-    "If you find the examples not running as expected. Please ensure you have built the entire AI-Powered Search project and are running it per the instructions at:\n",
-    "http://github.com/treygrainger/ai-powered-search\n"
+    "The book's code examples work with many different search engines and vector databases. By default, the book uses Apache Solr as the primary storage, matching, and ranking engine, but you can easily swap in your your favorite engine by running the `set_engine`"
```

Comment:
For all of these minor changes (non listing code) there is no value or use keeping the typo's or errors in the codebase other than consistency. I will always favor fixing the errors in the codebase instead of trying to maintain this perfect "slate". Even if there are 100 of these tiny, miniscule errors and we fix them, the book will still match 99.9999% of the book. I'll eat my hat if someone finds even 5 of these errors. No one is analyzing the book and the codebase for this type of consistency.

Reply:
I've fixed this typo and referenced the engine documentation.